borg/lrucache.py

```diff
@@ -26,7 +30,10 @@ def __delitem__(self, key):
         self._lru.remove(key)
 
     def __contains__(self, key):
-        return key in self._cache
+        contains = key in self._cache
+        self._hits += contains
+        self._misses += not contains
```

Comment:
I was about to say that client code would prefer `if x in cache: y = cache[x]`, but that's not the case for Repository. I'm just going to stash this, OT anyway.